test/integration/targets/postgresql/tasks/postgresql_privs.yml

```diff
@@ -1,56 +1,68 @@
----
-
-######################################################
-# Test foreign data wrapper and foreign server privs #
-######################################################
```

Comment:
Please, return the comment below where it was
```
######################################################
# Test foreign data wrapper and foreign server privs #
######################################################
```
And add something like before your test code
```
##########################################
# Test ALL_IN_SCHEMA for 'function' type #
##########################################
```
It'll be better to separate different tests